test/Test/Toml/Parsing/Property.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE ExplicitForAll      #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+module Test.Toml.Parsing.Property
+       ( propertyTests
+       ) where
+
+import Hedgehog (Property, forAll, property, (===))
+import Test.Tasty (TestTree)
+import Test.Tasty.Hedgehog (testProperty)
+
+import Toml.Parser (parse)
+import Toml.Printer (prettyToml)
+
+import Test.Toml.Gen (genToml)
+
+propertyTests :: [TestTree]
+propertyTests = [parsePrintTest]
+
+parsePrintTest :: TestTree
+parsePrintTest = testProperty "parse . prettyPrint == id" prop_parsePrint
```

Comment:
Maybe we can write `testProperty bla-bla-bla` right in `propertyTests` list to make such files shorter? :thinking: 
What do you think? Also no need to invent extra names.